src/tests/unit/tests/reports/components/inline-image.test.tsx

```diff
@@ -21,36 +21,32 @@ describe('InlineImageTest', () => {
 
     test('optional class name', () => {
         const testClassName = 'test-class-name';
-        const wrapped = shallow(
+        const renderResult = render(
             <InlineImage
                 imageType={InlineImageType.SleepingAda}
                 alt=""
                 className={testClassName}
             />,
         );
 
-        const element = wrapped.getElement();
-
-        expect(element.props.className).toEqual(testClassName);
+        expect(renderResult.container.getElementsByClassName(testClassName)).toBeTruthy();
     });
 
     function testInvalidImage(imageType: InlineImageType): void {
         const props: InlineImageProps = { imageType, alt: '' };
 
-        const wrapped = shallow(<InlineImage {...props} />);
+        const renderResult = render(<InlineImage {...props} />);
 
-        expect(wrapped.getElement()).toBeNull();
+        expect(renderResult.container.firstChild).toBeNull();
     }
 
     function testInlineImage(imageType: InlineImageType, alt: string): void {
         const props: InlineImageProps = { imageType, alt };
 
-        const wrapped = shallow(<InlineImage {...props} />);
-
-        const element = wrapped.getElement();
-
-        expect(element.type).toEqual('img');
-        expect(startsWith(element.props.src, 'data:image/png;base64,iVBO')).toBeTruthy();
-        expect(element.props.alt).toEqual(alt);
+        const renderResult = render(<InlineImage {...props} />);
+        expect(renderResult.getAllByRole('img')).toBeTruthy();
+        const element: any = renderResult.getByAltText(alt);
+        expect(element.src).toContain('data:image/png;base64,iVBO');
+        expect(element).toBeTruthy();
```

Comment:
I think we can make this a bit clearer by assigning the image element to a variable and testing that the image was rendered using the same method as the invalid test.
```suggestion
        const renderResult = render(<InlineImage {...props} />);
        expect(renderResult.container.firstChild).not.toBeNull();
        
        const element = renderResult.getByRole('img', { name: alt }) as HTMLImageElement;
        expect(element.src).toContain('data:image/png;base64,iVBO');
        expect(element.alt).toEqual(alt)
```

Reply:
Did the change as suggested.